cartographer/mapping_2d/sparse_pose_graph_test.cc

```diff
@@ -192,13 +197,14 @@ TEST_F(SparsePoseGraphTest, NoMovement) {
   MoveRelative(transform::Rigid2d::Identity());
   sparse_pose_graph_->RunFinalOptimization();
   const auto nodes = sparse_pose_graph_->GetTrajectoryNodes();
-  ASSERT_THAT(nodes.size(), ::testing::Eq(1u));
-  EXPECT_THAT(nodes[0].size(), ::testing::Eq(3u));
-  EXPECT_THAT(nodes[0][0].global_pose,
+  ASSERT_THAT(std::vector<int>{0},
+              ::testing::ContainerEq(ToVector(nodes.trajectory_ids())));
```

Comment:
Swap the vectors, i.e. actual value `nodes.trajectory_ids()` first, expected value in the matcher? Similar below.

Reply:
Done